src/blog/2024/04/displaying-logged-in-users-on-dashboard.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Displaying logged in user on Node-RED Dashboard 2.0
+subtitle: Step-by-Step Beginner's Guide to Displaying logged in User on Node-RED Dashboard 2.0
+description: Learn how to secure your Dashboard, install, and configure the Multi-user addon, and display logged-in users on Node-RED Dashboard 2.0. Additionally, delve deeper into understanding how the Multi-user addon functions.
+date: 2024-04-02
+authors: ["sumit-shinde"]
+image: 
+tags:
+   - posts
+   - dashboard 2.0
+
+---
+
+A few months ago when Dashboard 2.0 didn't exist, I was searching for customizable solutions with Dashboard 1.0 to build a more user-friendly, customizable, and powerful dashboard. Now Dashboard 2.0 is released, we've gained a more powerful, customizable, and well-maintained Dashboard compared to Dashboard 1.0. Now In this guide, we will explore one of the most powerful features that Dashboard 2.0 offers us. We will create a Multi-user dashboard that will display logged in user information on the dashboard.
+
+<!--more-->
+
+If you're new to Dashboard 2.0, refer to our blog post [Getting Started with Dashboard 2.0](https://flowfuse.com/blog/2024/03/dashboard-getting-started/)
+
+## Enabling Flowfuse User Authentication
+
+Before we can show user data on the dashboard we need to figure out how we get the data from users. This might seem complicated, but don’t worry about it. FlowFuse offers a feature that not only helps us secure the dashboard but also makes it simpler to get the user data at the dashboard at a later stage.
```

Comment:
```suggestion
Before we can show user data on the dashboard we need to figure out how we get the data from users. This might seem complicated, but don’t worry about it. FlowFuse offers a feature that not only helps us secure the dashboard but also makes it simpler to get the user data at the dashboard at a later stage.
```

Very long-winded way of saying _"FlowFuse makes it easy to secure your Node-RED Dashboards, and what is more, that authentication also provides us with contextual data in the Dashboard itself for whom is logged in."_

Reply:
We should also emphasise, probably _after_ the 3 steps are listed, the consequences of this change - users must now login in order to access the Dashboard.